pyproject.toml

```diff
@@ -54,7 +54,7 @@ numpy = ">=1.19.5, <2.0.0"
 pandas = ">=0.25.3, <2.0.0"
 scipy = "^1.5.4"
 tqdm = "^4.27.0"
-implicit = "0.4.4"
+implicit = "0.7.1"
```

Comment:
Let's maybe set "^0.7.1"?

Reply:
Should we start from 0.5.0 or 0.6.0 maybe? Tests required though

Reply:
I don't really think this makes sense

Reply:
fixed